plugins/CoreHome/vue/src/HelpLink/HelpLink.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <a :href="link" rel='noreferrer noopener' target='_blank'>
+  {{ text }}
+  </a>
+</template>
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  props: {
+    keyword: String,
+    base: String,
```

Comment:
I would use a computed property to process the input data.

Reply:
BTW we could maybe also simply hard code all the links without any component or something. Or at least in the JS. If we look at all the changes needed for this, and maintenance coming from it, might be easier to just add the parameters to the URL where needed and it's simple. If things ever change or needed somewhere else it's also easy to just add the parameters again since it's harder to figure out if there's a component for it etc.

Reply:
@sgiehl @peterhashair be good to think about if we can simplify all this so we also have to maintain less (component, tests, ...) and "know" (not needing to know there's a component for it) less. It also makes maybe things more clear. I know it was previously suggested to maybe use a component so sorry for sending another signal here but it's worth thinking about it. I'm not too much into it if it makes sense but all this complexity may not be needed for simply adding few campaign parameters that could be done in a few lines of code.

Reply:
My thought was that we actually don't want to have the additional parameters in the API response, as that could be used anywhere outside of Matomo. Also adjusting the links in every help text is a lot more and we need to keep that in mind every time we adjust or add a new one. Having a component makes that automatically and is easy to adjust later

Reply:
That could be still the case though without a component that we set the parameters in a central place? I would assume that we won't change anything there too much and when a change is needed, it be extremely quick to do. And it might be more likely that we first run into test issues with this where tests need to be fixed or the component eventually needs to be migrated again or we're making the campaign tracking more complex and then need to refactor it all again etc. Generally for such things be good to keep things simple.

Reply:
@tsteur So the idea is to do a global campaign variable, then append it to HTML href manually?

Reply:
I would have just put in the campaign parameters every time as a string, not even a global campaign variable. It's ultimately up to the team though. Just meaning it might make a lot more things simpler and faster

Reply:
@tsteur right, hardcode all the href params, update it now.

Reply:
I still have the opinion that a more global solution would have had more benefits. Like it would have been easier to adjust that later again. Or it would (if done right) e.g. have been possible to inject an additional tracking parameter on cloud, so we would have additional data how many people on cloud are clicking those links vs how many on self hosted.
Also there are a lot more links to documentation or faq spread across Matomo like in the diagnostic checks or in some no data or footer messages of reports. In addition the are more of such links in some of our plugins.

@tsteur feel free to go ahead and review and merge this PR if it sophisticates your needs.

Reply:
@peterhashair I would have eg expected to see the `plugins/Feedback/templates/index.twig` template changed. Can you go over the original issue and make sure that all the relevant articles are changed?

We'd also want to add a "medium" for some campaigns so we know on which link they clicked. 

Overall this looks a lot lot better though. Even if we had to change it eventually, it could be potentially changed with a search and replace. So far we don't really have any plans to change anything for cloud re the campaign parameters. If we plan to do that, then we can still see if there's a simple solution to it.

It must have been a lot faster to develop this way, easier to maintain and read, easier to review, easy to change and customise, etc.